### DISCUSSION TITLE
Add user to see specific sensor

### DISCUSSION THREAD

**Discussion by jarmouz** [Security-Onion-Solutions/securityonion#6900] [Unsupported Versions] [answered]
Hello,

I have installed  Security Onion with Distributed Architecture (one manager, mulitple sensors).
How can I add a user to see alerts for specific sensor?

Thank you

**Comment by dougburks** [comment, marked as answer]
https://docs.securityonion.net/en/2.3/rbac.html#creating-custom-roles
